client/views/pages/AccountActivate.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import DashboardBase from './DashboardBase';
+import Modal from '../common/Modal';
+
+/**
+ * DashboardPage for the /dashboard route
+ */
+const ActivatePage = () => (
```

Comment:
Appen Krashar.
![image](https://user-images.githubusercontent.com/9220635/42354580-1808fafc-80c8-11e8-895d-c3a0126c64b8.png)

Se: 
https://reactjs.org/docs/fragments.html
https://stackoverflow.com/questions/31284169/parse-error-adjacent-jsx-elements-must-be-wrapped-in-an-enclosing-tag
https://github.com/lan1dev/Playground-React-Redux/blob/master/src/views/components/ToDoApp.jsx